configure.ac

```diff
@@ -823,6 +823,25 @@ if test "x$HAVE_LIBDAEMON" = "xyes" ; then
 fi
 AM_CONDITIONAL(HAVE_LIBDAEMON, test "x$HAVE_LIBDAEMON" = "xyes")
 
+#
+# libsystemd
+#
+AC_ARG_ENABLE(libsystemd,
+        AS_HELP_STRING([--disable-libsystemd],[Disable use of libsystemd]),
+        [case "${enableval}" in
```

Comment:
I would prefer to default to auto value, which should use PKG_CHECK_MODULES to detect readiness of libsystemd development files. If detected, enable it. If not and ``${enablevar}`` were yes, emit fatal error.

Reply:
I followed the existing style of configure.ac, none of the (optional) library dependencies use `auto`, so it would be inconsistent if libsystemd was the only one doing that.